splink/linker.py

```diff
@@ -874,9 +875,27 @@ def compute_tf_table(self, column_name: str) -> SplinkDataFrame:
         input_col = InputColumn(column_name, settings_obj=self._settings_obj)
         tf_tablename = colname_to_tf_tablename(input_col)
         cache = self._intermediate_table_cache
+        concat_tf_tables = [
+            remove_quotes_from_identifiers(tf_col.input_name_as_tree).sql()
+            for tf_col in self._settings_obj._term_frequency_columns
+        ]
 
         if tf_tablename in cache:
             tf_df = cache[tf_tablename]
+        elif "__splink__df_concat_with_tf" in cache and column_name in concat_tf_tables:
+            self._pipeline.reset()
+            # If our df_concat_with_tf table already exists, use backwards inference to
+            # find a given tf table
+            sql = term_frequencies_from_concat_with_tf(InputColumn(column_name))
+            sql = {
+                "sql": sql,
+                "output_table_name": colname_to_tf_tablename(InputColumn(column_name)),
+            }
```

Comment:
Think this line is a little confusing, especially as both the SQL string variable and the wrapper dict are called `sql`. Can probably get rid of the dict altogether, seeing as it's just manually unpacked in the next line anyhow